sql/dupscore.sql

```diff
@@ -0,0 +1,23 @@
+CREATE TEMPORARY TABLE mytmp AS 
```

Comment:
where is this script being used?

Reply:
It's part of an upcoming feature to more easily identify and resolve duplicate patients. But doesn't fit with this PR, I'll remove it.